src/aiai_eval/utils.py

```diff
@@ -114,8 +116,6 @@ def log_scores(
     for metric_cfg in metric_configs:
         agg_scores = aggregate_scores(scores=scores, metric_config=metric_cfg)
         test_score, test_se = agg_scores["test"]
-        test_score *= 100
-        test_se *= 100
```

Comment:
Don't like scores of the form XX.XX%?

Reply:
It does not make sense to multiply the emissions/power metrics by 100. Thats why i removed them.

Reply:
Ah, fair enough!